apps/redi-connect/src/pages/app/applications/MobileView.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import Select from 'react-select'
+import { useSelector } from 'react-redux'
+
+import { RootState } from '../../../redux/types'
+import { Button } from '@talent-connect/shared-atomic-design-components'
+import { customStyles } from '../../../../../../libs/shared-atomic-design-components/src/lib/atoms/SelectStyles'
+import ApplicationCard from '../../../components/organisms/ApplicationCard'
+import './MobileView.scss'
+
+const applicationStatuses = [
+  { value: 'pending', label: 'Pending' },
+  { value: 'accepted', label: 'Accepted' },
+  { value: 'declined', label: 'Declined' },
+  { value: 'cancelled', label: 'Cancelled' },
+]
+
+const MobileView = ({
+  applicants,
+  filteredApplicants,
+  setActiveFilter,
+  activeFilter,
+}) => {
+  const profile = useSelector((state: RootState) => state.user.profile)
+  const isMentor = profile.userType === 'mentor'
+
+  const pendingApplications = applicants.filter(
+    (applicant) => applicant.status === 'applied'
+  )
+  const hasPendingApplications = Boolean(pendingApplications.length)
```

Comment:
Nice and declarative, these `has` variables. From the name it's also obvious they're all booleans.